NativeImage.jenkins

```diff
@@ -38,8 +38,11 @@ pipeline {
             label "win10"
           }
           steps {
-            powershell "if (Test-Path lemminx-win32.exe) { Remove-Item lemminx-win32.exe }"
-            powershell "Remove-Item -Recurse -Force lemminx"
+            powershell """
+              if (Test-Path lemminx-win32.exe) { Remove-Item lemminx-win32.exe }
+              Remove-Item -Recurse -Force lemminx
+              [Net.ServicePointManager]::SecurityProtocol
```

Comment:
This just outputs the https security protocol in use by powershell, right? If so, do we need this here?

Reply:
I can remove it. When I considered setting to Tls13, I decided it might be good to know what the builder is set as prior to changing it.